From: Daniel Baumann <daniel@debian.org>
Date: Sun, 8 Apr 2012 21:53:29 +0900
Subject: Customizes the help-dialog according to README.MODIFYING.

Description: Customizes the help-dialog according to README.MODIFYING.
 .
 * 2016-12-19 - Fixed path:
   /usr/share/common/licenses/GPL-2 -> /usr/share/common-licenses/GPL-2
   and restored use of "GPL" label.
Author: Daniel Baumann <daniel@debian.org>
Last-Update: 2016-12-19
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
---
 help-dialogs.c | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/help-dialogs.c b/help-dialogs.c
index c486613..758ac19 100644
--- a/help-dialogs.c
+++ b/help-dialogs.c
@@ -403,7 +403,7 @@ void ShowGPL()
   ShowTextfile(_("windowtitle|GNU General Public License"), 
 	       _("<big>GNU General Public License</big>\n"
 		 "<i>The license terms of dvdisaster.</i>"),
-	       "COPYING", NULL, NULL);
+	       "/usr/share/common-licenses/GPL-2", NULL, NULL);
 }
 
 /*
@@ -567,7 +567,7 @@ static void show_modifying(void)
 {  ShowTextfile(_("windowtitle|Modifying dvdisaster"), 
 	       _("<big>Modifying dvdisaster</big>\n"
 		 "<i>Your changes are not ours.</i>"),
-	       "README.MODIFYING", NULL, NULL);
+	       "/usr/share/doc/dvdisaster/README.MODIFYING", NULL, NULL);
 }
 
 static gint about_cb(GtkWidget *widget, GdkEvent *event, gpointer data)
@@ -746,11 +746,11 @@ void AboutDialog()
 			"GPL");
 
 #ifdef MODIFIED_SOURCE
-   AboutTextWithLink(vbox, _("\nThis program is <b>not the original</b>. It is based on the\n"
-			     "source code of dvdisaster, but contains third-party changes.\n\n"
+   AboutTextWithLink(vbox, _("\nThis version is <b>not the original</b>. It has been patched\n"
+			     "for Debian to support DVD-ROMs (with and without encryption).\n\n"
 			     "Please do not bother the original authors of dvdisaster\n"
-			     "([www.dvdisaster.org]) about issues with this version.\n"),
-		             "http://www.dvdisaster.org");
+			     "but submit bugreports against the [debian package] instead.\n"),
+			     "http://packages.qa.debian.org/dvdisaster");
 
 #else
    lang = g_getenv("LANG");
